phpcs.ruleset.xml

```diff
@@ -95,4 +95,7 @@
 	</rule>
 
 	<rule ref="SlevomatCodingStandard.Functions.StaticClosure" />
+
+	<!-- Exclude built plugins and built assets in plugins. -->
+	<exclude-pattern>./build/*</exclude-pattern>
```

Comment:
Nice. Should we also move `node_modules` and `vendor` here?

Reply:
I don't think so because those directories are exclusively in the root of the repo and aren't also in the plugin subdirectories.